torch/_inductor/fx_passes/post_grad.py

```diff
@@ -86,6 +87,8 @@ def post_grad_passes(gm: torch.fx.GraphModule, is_inference: bool):
 
     stable_topological_sort(gm.graph)
 
+    move_constructors_to_cuda(gm.graph)
```

Comment:
is fake tensor metadata up to date at this point? You need accurate device info right?

Reply:
we don't have any other device-changing optimizations so it should be..